robottelo/cli/base.py

```diff
@@ -194,12 +194,15 @@ def execute(cls, command, user=None, password=None, output_format=None,
         """Executes the cli ``command`` on the server via ssh"""
         user, password = cls._get_username_password(user, password)
 
-        cmd = u'LANG={0} hammer -v -u {1} -p {2} {3} {4}'.format(
+        # add time to measure hammer performance
+        perf_test = conf.properties['performance.test.foreman.perf']
+        cmd = u'LANG={0} {5} hammer -v -u {1} -p {2} {3} {4}'.format(
             conf.properties['main.locale'],
             user,
             password,
             u'--output={0}'.format(output_format) if output_format else u'',
-            command
+            command,
+            "time -p" if perf_test == 'true' else ''
```

Comment:
How about re-ordering these in a natural manner?

```python
        # add time to measure hammer performance
        perf_test = conf.properties['performance.test.foreman.perf']
        cmd = u'LANG={0} {1} hammer -v -u {2} -p {3} {4} {5}'.format(
            conf.properties['main.locale'],
            "time -p" if perf_test == 'true' else ''
            user,
            password,
            u'--output={0}'.format(output_format) if output_format else u'',
            command,
        )
```

There's two use cases for inserting indices in to `format()` calls:

1. You want to remain compatible with Python 2.6. Python 2.7 and above let you
   do this:

   ```python
   >>> '{} {}'.format('foo', 2)
   'foo 2'
   ```

   But Robottelo *generally* tries to be compatible with 2.6.

2. You want to repeat arguments. For example:

   ```python
   >>> '{0} {1} {0}'.format('foo', 2)
   'foo 2 foo'
   ```

In this case, we use indices to promote Python 2.6 compatibility and we are not
repeating arguments. So there's no reason to number things unnaturally.